EarlyBird/CoursesTableViewController.swift

```diff
@@ -24,4 +28,57 @@ class CoursesTableViewController: UITableViewController {
         cell!.textLabel!.text = courseNames[indexPath.row]
         return cell!
     }
+    
+    func makeGetRequest() {
+        // Set up the URL request for one specific course to before tring all courses
+        let courseEndpoint: String = "https://early-bird-courses.herokuapp.com/api/v1/courses/1"
```

Comment:
I think explicitly declaring the data type of `String` here is unnecessary, since you are supplying a value which is a string.